pkgs/servers/http/apache-httpd/2.4.nix

```diff
@@ -4,13 +4,15 @@
 , http2Support ? true, nghttp2
 , ldapSupport ? true, openldap
 , libxml2Support ? true, libxml2
+, brotliSupport ? true, brotli ? null
```

Comment:
what is the purpose of the `null` default?

Reply:
@zimbatm I copied this pattern from the curl default.nix: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/networking/curl/default.nix#L11
I figured being explicit about arguments was a good idea. I'm still learning nix so am open to better patterns, idiom etc.

Reply:
I see. Since nix is lazily-evaluated it's not a problem to always pass brotli. curl is a bit special as it is used during the stdenv bootstrap where these libraries aren't available at all.

Reply:
ah, thanks for the explanation. I'll PR to keep the style consistent.

Reply:
Or not. Thanks @orivej !